Header header
std_msgs/String color
std_msgs/String shape
sensor_msgs/Image img_file

